pkg/proc/eval.go

```diff
@@ -1183,7 +1183,8 @@ func (err *typeConvErr) Error() string {
 
 func (v *Variable) isType(typ godwarf.Type, kind reflect.Kind) error {
 	if v.DwarfType != nil {
-		if typ != nil && typ.String() != v.RealType.String() {
+		if typ == nil || !sameType(typ, v.RealType) {
+			fmt.Printf("%q %q\n", typ.String(), v.RealType.String()) //XXX
```

Comment:
What's the comment here?

Reply:
It means I wanted to delete that line before committing.